src/screens/HomeScreen.js

```diff
@@ -276,6 +277,8 @@ export const HomeScreen = ({ navigation, route }) => {
             />
 
             <Widgets widgetConfigs={widgetConfigs} />
+
+            <Disturber navigation={navigation} staticContentName="homeDisturber" />
```

Comment:
compared to the lines before for connected images carousel, what about using the same prop instead of `staticContentName`? 

```
<ConnectedImagesCarousel
  alternateAspectRatio
  navigation={navigation}
  publicJsonFile="homeCarousel"
  refreshTimeKey="publicJsonFile-homeCarousel"
/>

<Widgets widgetConfigs={widgetConfigs} />

<Disturber navigation={navigation} staticContentName="homeDisturber" />
```

do we use `staticContentName` already somewhere else?
how can we achieve the most aligned result for the code base?

Reply:
Yes, that's exactly right! I made this change to improve the understandability of the code. Now the components are more similar to each other.

https://github.com/smart-village-solutions/smart-village-app-app/pull/909/commits/2852592d533e68dc4016d32fb7c84d40c0faee51